### PR TITLE
Fix typos in project

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `typos --format brief`.

### CODE REVIEW

**Comment by ysfscream** [comment]
Thanks for your contribution! @kianmeng

**Review by ysfscream — APPROVED** [review]

File: src/i18n/Auth.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ export default {
     zh: '测试通过，您的配置可以正常使用',
     en: 'Test success, your configuration works fine',
   },
-  testFaild: {
+  testFailed: {
```
Comment (Kinplemelon) [inline src/i18n/Auth.js:122]: mark

File: src/i18n/Base.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ export default {
     zh: '资源未找到',
     en: 'Resources not found',
   },
-  loging: {
+  logging: {
```
Comment (Kinplemelon) [inline src/i18n/Base.js:30]: this is useless, can delete it

File: src/i18n/Clients.js [hunk 3]
```diff
@@ -387,7 +387,7 @@ export default {
     zh: 'Retain Handling',
     en: 'Retain Handling',
   },
-  topicRequried: {
+  topicRequired: {
```
Comment (Kinplemelon) [inline src/i18n/Clients.js:390]: mark

File: src/i18n/Dashboard.js [hunk 4]
```diff
@@ -327,7 +327,7 @@ export default {
     zh: '不再提示',
     en: 'Stop showing this',
   },
-  konw: {
+  know: {
```
Comment (Kinplemelon) [inline src/i18n/Dashboard.js:330]: mark

File: src/i18n/Auth.js (hunk 1 above)
Comment (Kinplemelon) [inline src/i18n/Auth.js:122]: mark

**Review by Kinplemelon — APPROVED** [review]

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**